src/components/shared/About/privacy-policy/PrivacyPolicy.module.css

```diff
@@ -0,0 +1,65 @@
+.model_container {
+  display: flex;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #00000080;
+  z-index: 550;
+}
+
+.model_body {
+  width: 500px;
+  max-width: 500px;
+  height: auto;
+  background-color: #fff;
+  border-radius: 10px;
+  box-shadow: 0 0 10px #0003;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  z-index: 600;
+  cursor: crosshair;
+  overflow-y: auto;
+}
+
+.model_event {
+  width: 100%;
+  height: 100%;
+  position: absolute;
```

Comment:
is absolute necessary?

Reply:
Yup, otherwise it would be positioned over the sidebar.  We'll most certainly revisit when we get to testing the overall responsiveness of the app.